tests/functional/d/dict_iter_missing_items.py

```diff
@@ -1,4 +1,5 @@
 # pylint: disable=invalid-name, consider-iterating-dictionary, missing-docstring, import-error
+# pylint: disable=unbalanced-dict-unpacking
```

Comment:
Where is the new unbalanced-dict-unpacking ? (Iine 12 I suppose ?). We try to not raise 2 messages for the same problem but we might want to be explicit if we choose to do it here. Or we may not raise it in this specific case because we already raise a more accurate message. Maybe it's the occasion to merge the two checks and optimize the one we're merging in the other ?

```suggestion
```

Reply:
Yeah, line 12 yields that warning because we're unpacking the keys of a dict onto 2 variables.

As far as I can tell, there's no case where `dict-items-missing-iter` is raised and `unbalanced-dict-unpacking` isn't, since the check for missing `items` is implemented by looking for unbalanced dict unpacking - It looks for `for` loops with 2 targets while iterating a dict where keys aren't tuples. 

Maybe `dict-items-missing-iter` is deprecated at this point.

Reply:
Yeah, line 12 yields that warning because we're unpacking the keys of a dict onto 2 variables.

As far as I can tell, there's no case where `dict-items-missing-iter` is raised and `unbalanced-dict-unpacking` isn't, since the check for missing `items` is implemented by looking for unbalanced dict unpacking - It looks for `for` loops with 2 targets while iterating a dict where keys aren't tuples. 

Maybe `dict-items-missing-iter` is deprecated at this point.

Reply:
``dict-items-missing-iter`` is a more precise message than ``unbalanced-dict-unpacking`` (it tells precisely what the solution is to fix the issue) so even if it would make sense for us to remove it to make our code leaner, I don't think that's the best solution from a user perspective.

Reply:
OK, yeah, that makes sense. I'll try and detect whether the conditions for `dict-items-missing-iter` hold when checking for `unbalanced-dict-unpacking` warning and omit the latter warning if they do.

Reply:
Sounds good !

Reply:
Implemented. Let me know if there's what you had in mind